### ISSUE TITLE
Could not find configuration node: databases/database[@id='master']

### ISSUE THREAD

**Issue by kamsar** [sergeyshushlyapin/Sitecore.FakeDb#194]
Hey Sergey,

I just installed FakeDb on SC9 RTM to do some tests. My project is a .NET Core-style project (with the new .csproj format, no packages.config). It's targeting .NET 4.6.2, just using the new format.

This means that it could not install the `app.config` by default (nuget 3 cannot install content items to the project); so I installed it manually. I also referenced all the assemblies, and have tried about 15 different license paths.

But no matter what, as soon as I run `new Db()`, I get this:
```
System.InvalidOperationException : Could not find configuration node: databases/database[@id='master']
   at Sitecore.Configuration.DefaultFactory.GetConfigNode(String xpath, Boolean assert)
   at Sitecore.Configuration.DefaultFactory.CreateObject(String configPath, String[] parameters, Boolean assert)
   at Sitecore.Configuration.DefaultFactory.GetDatabase(String name, Boolean assert)
   at Sitecore.Configuration.DefaultFactory.GetDatabase(String name)
   at Sitecore.FakeDb.Db..ctor(String databaseName)
```

Making the license path intentionally fake doesn't seem to change anything; but the test dll does have a corresponding `.config` next to it with the FakeDb settings in it.

Any ideas?

**Comment by kamsar** [comment]
fwiw, I also have `<sc.variable name="databaseType" value="Sitecore.Data.DefaultDatabase, Sitecore.Kernel" />` in the config for 8.2+

**Comment by kamsar** [comment]
complete app.config for reference
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <configSections>
        <section name="sitecore" type="Sitecore.FakeDb.Configuration.ConfigReader, Sitecore.FakeDb" />
        <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, Sitecore.Logging" />
    </configSections>
  <runtime>
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <dependentAssembly>
        <assemblyIdentity name="System.Reactive.Linq" publicKeyToken="94bc3704cddfc263" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-3.0.3000.0" newVersion="3.0.3000.0" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Threading.Tasks.Extensions" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.1.1.0" />
      </dependentAssembly>
    </assemblyBinding>
  </runtime>
    <appSettings>
        <add key="xunit.parallelizeTestCollections" value="false" />
    </appSettings>
    <sitecore>
        <settings>
            <setting name="LicenseFile" value="/license.xml" />
        </settings>
        <!-- DATABASE TYPE
         For Sitecore versions prior to 8.2 should be 'Sitecore.Data.Database, Sitecore.Kernel'.
         For Sitecore 8.2 and later should be 'Sitecore.Data.DefaultDatabase, Sitecore.Kernel'. -->
        <sc.variable name="databaseType" value="Sitecore.Data.DefaultDatabase, Sitecore.Kernel" />
    </sitecore>
    <log4net />
    <system.web>
        <membership defaultProvider="fake">
            <providers>
                <clear />
                <add name="fake" type="Sitecore.FakeDb.Security.Web.FakeMembershipProvider, Sitecore.FakeDb" />
            </providers>
        </membership>
        <roleManager defaultProvider="fake" enabled="true">
            <providers>
                <clear />
                <add name="fake" type="Sitecore.FakeDb.Security.Web.FakeRoleProvider, Sitecore.FakeDb" />
            </providers>
        </roleManager>
        <profile defaultProvider="fake" enabled="true" inherits="Sitecore.FakeDb.Profile.FakeUserProfile, Sitecore.FakeDb">
            <providers>
                <clear />
                <add name="fake" type="Sitecore.FakeDb.Security.Web.FakeProfileProvider, Sitecore.FakeDb" />
            </providers>
            <properties>
                <clear />
                <add type="System.String" name="SC_UserData" />
            </properties>
        </profile>
    </system.web>
</configuration>
```

**Comment by kamsar** [comment]
I figured something out. This whole thing started when the Resharper test runner crashed before even executing my FakeDb test. So I tried the nunit console runner and got the above issue.

Now I've restarted my computer and the R# test runner seems to be just fine with FakeDb, though the console runner is still complaining about the database node. Not sure why it'd have issues deserializing the embedded resource config file (great idea, btw) when run from the console.

Closing this, as I doubt it's a FakeDb issue directly.
